crates/cadenza-eval/src/ir/generator.rs

```diff
@@ -300,6 +301,18 @@ impl IrGenerator {
             "measure" => special_form::measure_form::get(),
             "|>" => special_form::pipeline_form::get(),
             "." => special_form::field_access_form::get(),
+            "+" => special_form::add_form::get(),
+            "-" => special_form::sub_form::get(),
+            "*" => special_form::mul_form::get(),
+            "/" => special_form::div_form::get(),
+            "==" => special_form::eq_form::get(),
+            "!=" => special_form::ne_form::get(),
+            "<" => special_form::lt_form::get(),
+            "<=" => special_form::le_form::get(),
+            ">" => special_form::gt_form::get(),
+            ">=" => special_form::ge_form::get(),
+            "&&" => special_form::and_form::get(),
+            "||" => special_form::or_form::get(),
```

Comment:
Can you make a note in the status doc that we should look up special forms from the evaluator environment rather than hardcoding a mapping. This will make it easier to maintain and extend the language and make it more consistent with the interpreter.